dockers/docker-database/database_config.json.j2

```diff
@@ -93,6 +93,11 @@
             "id" : 14,
             "separator": ":",
             "instance" : "redis"
+        },
+        "PROFILE_DB" : {
```

Comment:
PROFILE_DB is similar to CONFIG_DB. As a general practice, we prefer `|` as separator for newly added DB index.

Reply:
Fixed, also will fix config files in swss-common PR later.